### DISCUSSION TITLE
How to find a build that contains the internal PDF viewer?

### DISCUSSION THREAD

**Discussion by holgerflick** [texstudio-org/texstudio#2979] [Q&A] [answered]
The most recent release 4.5.1 keeps "jumping" in the code editor and i want to find out if it is something that has been fixed in the meantime. However, no matter which build I download, they never contain the internal PDF viewer. Is there any way to "see" if the internal PDF viewer is included or not?

Thanks!

**Comment by sunderme** [comment]
all builds here contain the pdf viewer.

**Reply by holgerflick** [reply]
Whenever I enable the PDF viewer I get the message that the build has been built without the internal PDF viewer. MacOS. Today's master.

**Comment by muzimuzhi** [comment]
What's your macOS version?

**Reply by holgerflick** [reply]
`13.2.1 (22D68)`

**Comment by sunderme** [comment, marked as answer]
fixed

**Reply by holgerflick** [reply]
thanks. can I test the latest checkin on a certain branch or did you merge into master?

**Reply by holgerflick** [reply]
found it and confirmed it works.
